src/dmd/backend/mach.d

```diff
@@ -404,3 +405,28 @@ struct scattered_relocation_info
 
     int r_value;
 }
+
+struct build_version_command
+{
+    uint cmd = LC_BUILD_VERSION;
+    uint cmdsize = build_version_command.sizeof;
+
+    uint platform = PLATFORM_MACOS;
+    uint minos = 659200; // 10.15
+    uint sdk = 659200; // 10.15
```

Comment:
These three attributes can be configured when compiling with Clang. Is this something we want to support? By default it will use the current platform and current SDK to set these values.